collectors/common.py

```diff
@@ -2,14 +2,40 @@
 
 
 import socket
-from os import statvfs
-import math
+import os
+import subprocess
+
+
+def cmd_exists(command):
+    return any(
+        os.access(os.path.join(path, command), os.X_OK)
+        for path in os.environ["PATH"].split(os.pathsep)
+    )
+
+
+def os_cmd(command):
+    """
+    Issue a command to the OS and return the output. NB. check_output default
+    is shell=False
+    :param command: (str) OS command
+    :return: (list) command response
+    """
+    cmd_list = command.split(' ')
+    if cmd_exists(cmd_list[0]):
+        cmd_output = subprocess.check_output(cmd_list).rstrip()
+        return cmd_output
+    else:
+        return ''
 
 
 def get_hostname():
     return socket.gethostname().split('.')[0]
 
 
+def get_names():
+    return [get_hostname()]
```

Comment:
I wanted to leave the door open for a mon to have multiple names - the current function just yields one, but it could be changed to provide IP addrs and names...just positioning really.